frontend/src/app/app.module.ts

```diff
@@ -3,12 +3,30 @@ import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
 
+import {AccordionModule, MenuModule, SidebarModule} from 'primeng/primeng';
+import { AppRoutingModule } from './/app-routing.module';
+import { HomeComponentComponent } from './home-component/home-component.component';
+import { FaqComponentComponent } from './faq-component/faq-component.component';
+import { HowtouseComponentComponent } from './howtouse-component/howtouse-component.component';
+import { HeaderComponent } from './header/header.component';
+import { FooterComponent } from './footer/footer.component';
+
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
```

Comment:
Please pay attention to abc sequence. It is easier to find components and others.